package.json

```diff
@@ -45,7 +45,8 @@
     "redis": "1.0.0",
     "relations": "0.5.0",
     "uuid": "2.0.1",
-    "vision": "3.0.0"
+    "vision": "3.0.0",
+    "boom": "2.8.0"
```

Comment:
Boom is already present on [line 16](https://github.com/chrisfilo/nodeapi/blob/develop/package.json#L16)